src/Illuminate/Queue/Jobs/RedisJob.php

```diff
@@ -17,13 +17,20 @@ class RedisJob extends Job implements JobContract
     protected $redis;
 
     /**
-     * The Redis job payload.
+     * The Redis raw job payload.
      *
      * @var string
      */
     protected $job;
 
     /**
+     * The Redis decoded job payload.
+     *
+     * @var array
+     */
+    protected $decodedJob;
```

Comment:
yeah, i think this wil be nice) but Taylor already merge this pr. Create another with renaming?